tests/perf_helper.py

```diff
@@ -1,3 +1,7 @@
+import cProfile
+import os
+import pstats
+import sys
```

Comment:
nicee - tyty, was this done manually or via ruff?

Reply:
Manually for this one